test/flint-example-lerarenbeurs.json

```diff
@@ -959,7 +959,10 @@
     },
     {
       "fact": "[besluit]",
-      "function": { "expression": "CREATE" },
+      "function": {
+        "expression": "CREATE",
+        "operands": []
```

Comment:
Do you think we should make operands optional? I think the trade-off here is that we don't have this optionality in other places, but it avoids introducing a breaking change.

Reply:
I agree. Also, the expression it useful without any operands. So it makes sense to me